web/sites/default/config/block.block.messages.yml

```diff
@@ -1,22 +1,20 @@
-uuid: 751f2e55-ca80-403f-8138-ca6ae6588c13
+uuid: 17c7f455-20f6-4ee2-8d1f-95270a65d0e4
 langcode: en
 status: true
 dependencies:
   module:
     - system
   theme:
     - cloudy
-_core:
-  default_config_hash: 9RnnTUBcn0WRWXfrGOnId2H0Q31uQUeu_F5TnDxl2yg
-id: cloudy_messages
+id: messages
```

Comment:
When I added back status messages, it was not ideal that this changed as it didn't need to, but it doesn't hurt anything as we don't use the id cloudy_messages in our custom code or theme.